planning/route_handler/src/route_handler.cpp

```diff
@@ -433,6 +434,10 @@ Pose RouteHandler::getStartPose() const
 
 Pose RouteHandler::getOriginalStartPose() const
 {
+  if (!route_ptr_) {
+    RCLCPP_WARN(logger_, "[Route Handler] getOriginalStartPose: Route has not been set yet");
+    Pose();
```

Comment:
Is `return Pose()` correct?
(Maybe, we have to change other lines in the same way)

Reply:
It is not good but I just copy from other part like
https://github.com/autowarefoundation/autoware.universe/blob/5094fa26085bc90a3ad943c6680a093b8cddb61d/planning/route_handler/src/route_handler.cpp#L439-L446

```
Pose RouteHandler::getGoalPose() const
{
  if (!route_ptr_) {
    RCLCPP_WARN(logger_, "[Route Handler] getGoalPose: Route has not been set yet");
    Pose();
  }
  return route_ptr_->goal_pose;
}
```

Reply:
oh I misunderstood. fixed in @kosuke55
[fix return](https://github.com/autowarefoundation/autoware.universe/pull/4876/commits/fe4a4f127494081d6f0e343b1c7ad91f768854e3)